### DISCUSSION TITLE
Pade doesn't show in the Openfire 4.6.1 Available Plugins

### DISCUSSION THREAD

**Discussion by abel406** [igniterealtime/openfire-pade-plugin#234] [General]
Hello and thanks to the Openfire Team for the good job.
Does anybody agree with the idea that Pade plugin should appear in the Openfire 4.6.1 Available Plugins ?

**Comment by gjaekel** [comment]
Dear Abel,
thank you for your remark! Since I'm involved in development, I always download from the release page and install by hand :)

@deleolajide : bug or intention?

**Reply by abel406** [reply]
Thank you for the quick reply @gjaekel . I say this becouse i had a hardtime getting Openfire Meetings plugin to work. After reading some logs I realized that it was a connection issue. I live in a Country Blocked by some US stupid laws and i had to use VPN to get the jar files downloaded from Amazon's servers. Once i did that, Meetings installed smoothly but Pade didn't. I tryed version 1.2.2 and 1.3.0 of Pade with no luck. So i thought that maybe it could be included in the mentioned list with some compatibility and requirements checks prior download and installation.

**Reply by gjaekel** [reply]
There was a "unification" recently:
* **OFFocus** -- providing the JiCoFo component -- was integrated in OFMeet, first 
* **OFMeet** -- providing the enhanced OpenMeeting Webclient and the Jitsi Vido Bridge (JVB/JVB2) and the JiGaSi SIP Gateway -- was integrated in Pàdé, second
* **Pàdé** now provides as well the enhanced Converse/Inverse XMPP text chat client as the OFmeet/Jitsi audio/video chat. You may just use the A/V part "stand-alone" or may use Pàdé to provide integration and cross-functionality.

That happens with the intention to _avoid_ dealing with different and maybe incompatible plugin packages. If you install the recent Pàdé plugin, you **must** **not** install any other.

> So i thought that maybe it could be included in the mentioned list with some compatibility and requirements checks prior download and installation.

You're welcome warm to support the sub-project by enhancing the documentation!

**Reply by abel406** [reply]
I tried Pade latest version without OFFocus and OFMeet, and it didn't worked. I am using Windows Server 2019 enterprise, PostgreSQL 12 and Openfire 4.6.1 for 64 bits.
Regading the documentation I would love to, in my so little spare time.

**Reply by deleolajide** [reply]
There is an issue with pade plugin and the latest openfire 4.6.1 release. Try the openfire nightly builds or downgrade to 4.6.0.
Someone else has reported this issue, but I can't find the link to it.

**Reply by gjaekel** [reply]
@deleolajide To you refer to this the thing with a non-appearing tab in the Admin UI?

**Reply by deleolajide** [reply]
Yes. That bug

**Reply by gjaekel** [reply]
It's #209.

Meta: I search about, but it seems that in the moment one can't convert a Discussion into an Issue (see https://github.com/github/feedback/discussions/2861). But I think it's better to open a Issue if @abel406 is not able to solve this in an easy way.

**Reply by deleolajide** [reply]
OF 4.6.1 has made some breaking API changes to Openfire MUC implementation. I am hoping to tackle that with a 1.4.0 release next month that will only be compatible with OF 4.6.1+

**Reply by gjaekel** [reply]
@abel406 I may provide you the tarball of the openfire-4.6.1 I'm currently using. This seems to work well in my eyes.

**Reply by deleolajide** [reply]
Pade now builds to archiva like all official plugins

https://igniterealtime.org/projects/openfire/plugin-archive.jsp?plugin=pade
